action.yml

```diff
@@ -2,52 +2,51 @@ name: Update Deployment Board
 description: Adds deployment Issues to an Automated Deployment Project Board for visual tracking of what has been deployed to each environment.
 
 inputs:
-  github-token:
-    description: 'A token with permissions to create and update issues'
+  workflow-actor:
+    description: The GitHub user who triggered the workflow
     required: true
-  github-login:
-    description: 'The login associated with the github-token.  Defaults to github-actions'
+  token:
+    description: A token with permissions to create and update issues
     required: true
   environment:
-    description: 'The environment the branch, tag or SHA was deployed to'
+    description: The environment the branch, tag or SHA was deployed to, i.e. [DEV|QA|STAGE|DEMO|UAT|PROD]
     required: true
-  board-number:
-    description: 'The number of the project board that will be updated.  Found by using the number in the board url.'
+  project-slug:
```

Comment:
The term `project-slug` is a Backstage term. It would probably be clearer here if this parameter was named `repository` or something like that.

But I'm surprised the previous version of this action didn't need such a parameter. Why didn't it need one, and can this version work without it?